internal/infrastructure/db/migrations/20231012142752_add_msg_col_position.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+ALTER TABLE pid_configs ADD COLUMN message_column_position INT;
+UPDATE pid_configs SET message_column_position = 0 WHERE message_column_position IS NULL;
```

Comment:
instead of this line, you can just use the default option in line above.
https://www.postgresql.org/docs/current/ddl-default.html